adal/src/androidTest/java/com/microsoft/aad/adal/AcquireTokenRequestTest.java

```diff
@@ -448,6 +448,47 @@ private ITokenCacheStore getTokenCache(final Date expiresOn, final boolean store
         return cacheStore;
     }
 
+    private ITokenCacheStore getExtendedLifetimeTokenCache(final Date expiresOn, final boolean storeMRRT, final boolean storeFRT, final Date extendedExpiresOn) {
```

Comment:
The current tests file already has code to return tokencache store, I would prefer you changing existing method or thinking of ways to refactor to reduce duplicate code.